api/types/network/network.go

```diff
@@ -1,97 +1,102 @@
-// Code generated by go-swagger; DO NOT EDIT.
+// GENERATED CODE; DO NOT EDIT
 
 package network
 
-// This file was generated by the swagger tool.
-// Editing this file might prove futile when you re-run the swagger generate command
-
 import (
-	timeext "time"
+	"time"
 )
 
-// Network network
-//
-// swagger:model Network
+// Network
 type Network struct {
-
 	// Name of the network.
 	//
 	// Example: my_network
-	Name string `json:"Name"`
+	Name string `json:"Name,omitempty"`
 
 	// ID that uniquely identifies a network on a single machine.
 	//
 	// Example: 7d86d31b1478e7cca9ebed7e73aa0fdeec46c5ca29497431d3007d2d9e15ed99
-	ID string `json:"Id"`
+	ID string `json:"Id,omitempty"`
 
 	// Date and time at which the network was created in
 	// [RFC 3339](https://www.ietf.org/rfc/rfc3339.txt) format with nano-seconds.
 	//
 	// Example: 2016-10-19T04:33:30.360899459Z
-	Created timeext.Time `json:"Created"`
+	Created time.Time `json:"Created,omitempty"`
 
 	// The level at which the network exists (e.g. `swarm` for cluster-wide
 	// or `local` for machine level)
 	//
 	// Example: local
-	Scope string `json:"Scope"`
+	Scope string `json:"Scope,omitempty"`
 
 	// The name of the driver used to create the network (e.g. `bridge`,
 	// `overlay`).
 	//
 	// Example: overlay
-	Driver string `json:"Driver"`
+	Driver string `json:"Driver,omitempty"`
 
 	// Whether the network was created with IPv4 enabled.
 	//
 	// Example: true
-	EnableIPv4 bool `json:"EnableIPv4"`
+	EnableIPv4 bool `json:"EnableIPv4,omitempty"`
 
 	// Whether the network was created with IPv6 enabled.
 	//
 	// Example: false
-	EnableIPv6 bool `json:"EnableIPv6"`
+	EnableIPv6 bool `json:"EnableIPv6,omitempty"`
 
-	// The network's IP Address Management.
 	//
-	IPAM IPAM `json:"IPAM"`
+	IPAM *IPAM `json:"IPAM,omitempty"`
```

Comment:
This changed to a pointer; is there a way to control that through the swagger?

Reply:
sure. A pointer is used when attribute isn't required